dev/details.html

```diff
@@ -9,13 +9,15 @@
 
     <script type="module">
       import '@vaadin/details';
+      import '@vaadin/tooltip';
     </script>
   </head>
 
   <body>
     <vaadin-details>
       <div slot="summary">Expandable Details</div>
       Toggle using mouse, Enter and Space keys.
+      <vaadin-tooltip slot="tooltip" text="Click to open"></vaadin-tooltip>
```

Comment:
nit: to avoid
![Screenshot 2022-09-01 at 17 14 50](https://user-images.githubusercontent.com/1222264/187936219-91cd5e3c-c3e8-42fb-b67e-ee62271968a2.png)

```suggestion
      <vaadin-tooltip slot="tooltip" text="Click to toggle"></vaadin-tooltip>
```